src/R/lib_size_sc_V5_single_species.R

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/Rscript
 
 args <- commandArgs()
-elbow_finder <- function(x_values, y_values) {
+elbow_knee_finder <- function(x_values, y_values) {
```

Comment:
I wrote it down but I found new things and I am going to add that you need to really explain what the inputs are. I noticed that you input the x linear but the y log10. Does it have to work this way?  Can you pass both linear? Explain what you expect to see. Even better would be to always expect linear and do the log inside